src/CodeFormatter.cs

```diff
@@ -29,27 +29,27 @@ internal static class CodeFormatter
             new EndOfLineFormatter(),
             new CharsetFormatter(),
             new ImportsFormatter(),
-            new AnalyzerFormatter(new RoslynCodeStyleAnalyzerFinder(), new AnalyzerRunner(), new SolutionCodeFixApplier()),
+            new AnalyzerFormatter("Code Style", new RoslynCodeStyleAnalyzerFinder(), new AnalyzerRunner(), new SolutionCodeFixApplier()),
```

Comment:
Can we add constants for these strings?

Reply:
Will do in a followup pr. likely want to change how these formatters are instantiated regardless.